### PR TITLE
PR: Filters

### PR DESCRIPTION
**Pull request by pavan6115**
- Added New Products
- Wrote code for filtering products as per categories
- Added Routes

### CODE REVIEW

File: src/pages/Cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import './Cart.css'
+import { Link } from "react-router-dom"
+// import carBrushes from '../../assets/car-brushes.png'
+// import microFiberCloth from '../../assets/car-microfiber-cloth.png'
+// import carWashShampoo from '../../assets/car-wash-shampoo.png'
+// import engineOil2 from '../../assets/engine-oil-2.png'
```
Comment (viru2001) [inline src/pages/Cart/Cart.jsx]: you can remove not used code

File: src/pages/Wishlist/WishList.jsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import React from 'react'
+import './WishList.css'
+import { Link } from 'react-router-dom'
+// import microFiberCloth from '../../assets/car-microfiber-cloth.png'
+// import engineOil2 from '../../assets/engine-oil-2.png'
+// import chassis from '../../assets/chassis.png'
```
Comment (viru2001) [inline src/pages/Wishlist/WishList.jsx]: remove this also

**Review by viru2001 — APPROVED** [review]
looks good to merge

File: src/pages/Cart/Cart.jsx (hunk 1 above)
Comment (pavan6115) [reply src/pages/Cart/Cart.jsx]: okay, will do 👍

File: src/pages/Wishlist/WishList.jsx (hunk 2 above)
Comment (pavan6115) [reply src/pages/Wishlist/WishList.jsx]: yes 👍

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+:root {
+  --background-color: #f1f5f9;
+  --primary-color: #F28627;
+  --secondary-color: rgb(242, 92, 5);
+  --border-color: rgb(140, 28, 3);
+  --white-color: #ffffff;
+  --slight-gray: #B5B5B5; 
+}
```
Comment (JuvitaSaini09) [inline src/App.css:16]: Suggestion: you can variable for colors

**Review by JuvitaSaini09 — APPROVED** [review]
To me your code looks good to merge

File: src/App.css (hunk 3 above)
Comment (pavan6115) [reply src/App.css:16]: okay 👍

**Pull request merged**